CHANGES.rst

```diff
@@ -4,6 +4,9 @@ ChangeLog
 +------------+---------------------------------------------------------------------+------------+
 | Version    | Description                                                         | Date       |
 +============+=====================================================================+============+
+| **1.16.2** | * Added new parallel interface module                               | 2020/09/20 |
```

Comment:
replace release date with TBA, it'll be updated day it's going on pypi

Reply:
Changed as suggested.  Will do the same with ``luma.lcd`` and ``luma.oled``